internal/captain/command.go

```diff
@@ -237,9 +237,12 @@ func (c *Command) AddLegacyChildren(children ...cobraCommander) {
 func (c *Command) Find(args []string) (*Command, error) {
 	foundCobra, _, err := c.cobra.Find(args)
 	if err != nil {
-		return nil, err
+		return nil, locale.WrapError(err, "err_captain_cobra_find", fmt.Sprintf("Could not find child Command with args: %s", args))
```

Comment:
`fmt.Sprintf` is working around the locale mechanic. This should be something like this:

```
return nil, locale.WrapError(err, "err_captain_cobra_find", "Could not find child Command with args: {{.V0}}", args)
```

But this probably shouldn't be a localized error at all.